picturae_importer.py

```diff
@@ -357,6 +357,11 @@ def create_agent_list(self, row):
                 if agent_id is None:
                     self.new_collector_list.append(collector_dict)
 
```

Comment:
fills in empty agent dics with a placeholder string as per botany staff requests.